types/koa/index.d.ts

```diff
@@ -456,7 +456,14 @@ declare class Application<
     silent: boolean;
     keys: Keygrip | string[];
 
-    constructor();
+    constructor(options?: {
```

Comment:
Koa offers some nice jsdoc https://github.com/koajs/koa/blob/master/lib/application.js#L39

Maybe it's worth adding it here too?

Reply:
Year, I add it together